pkgs/applications/window-managers/qtile/wrapper.nix

```
@@ -0,0 +1,9 @@
+{ python3, qtile-unwrapped }:
+(python3.withPackages (_: [ qtile-unwrapped ])).overrideAttrs (_: {
+  # otherwise will be exported as "env", this restores `nix search` behavior
+  name = "${qtile-unwrapped.pname}-${qtile-unwrapped.version}";
+  # export underlying qtile package
+  passthru = { inherit qtile-unwrapped; };
```

```suggestion
  passthru = { unwrapped = qtile-unwrapped; };
```

Just in case someone depended on `qtile.unwrapped` existing rather than `qtile.qtile-unwrapped`, it might be best to avoid changing this name here